app/views/account_cards/duty_deferment_inaccurate_balances_message.scala.html

```diff
@@ -26,6 +26,6 @@ <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
         <span class="govuk-warning-text__assistive">@messages("cf.duty-deferment.outOfDateBalance.hiddenText")</span></strong>
     <p class="govuk-body govuk-!-margin-top-4">@messages("cf.duty-deferment.outOfDateBalance.chiefText")</p>
     <p class="govuk-body govuk-!-margin-bottom-0">@messages("cf.duty-deferment.outOfDateBalance.getLatestBalance")
-        <a target="_blank" rel="noreferrer noopener" href="@(appConfig.dutyDefermentSchemeContactLink)" class="govuk-link">@messages("cf.duty-deferment.outOfDateBalance.contactLink")</a>
+        <a target="_blank" rel="noreferrer noopener" href="@(appConfig.dutyDefermentSchemeContactLink)" class="govuk-link">@messages("cf.duty-deferment.outOfDateBalance.contactLink")</a>@messages("cf.duty-deferment.outOfDateBalance.getLatestBalanceFullStop")
```

Comment:
Confused - a previous ticket was to ensure there were no fullstops at the end of links!
From GDS:

![image](https://user-images.githubusercontent.com/35488891/146398514-6f5b2731-d318-4379-9e89-e9bf0807ccff.png)

Reply:
Hi Derek, :slightly_smiling_face:
During the meeting we had after standup today, it was advice that we put full stop after the Link (not within the link). They wanted full stop at the end of the paragraph, hence this PR
I hope this makes sense :slightly_smiling_face: